vertical-pod-autoscaler/pkg/recommender/util/binary_decaying_histogram_test.go

```diff
@@ -43,9 +43,8 @@ func bucketLength(options HistogramOptions, value float64) float64 {
 	// special handling of the last bucket
 	if bucket == options.NumBuckets()-1 {
 		return options.GetBucketStart(bucket) - options.GetBucketStart(bucket-1)
-	} else {
-		return options.GetBucketStart(bucket+1) - options.GetBucketStart(bucket)
 	}
+	return options.GetBucketStart(bucket+1) - options.GetBucketStart(bucket)
```

Comment:
Was this pointed out by linter?
Personally I like the original version better but I have no strong opinions.

Reply:
Yes pointed out by linter